google/auth/_oauth2client_async.py

```diff
@@ -0,0 +1,171 @@
+# Copyright 2016 Google LLC
```

Comment:
I don't see anything async-specific in this file. Could you speak to why it was necessary to duplicate this?

Reply:
This new file was made to run the oauth2client code with the async service account functions.